SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -306,7 +306,7 @@ EventBug = 1
 # Показ миникарты во время хода ИИ, при условии выключенной опции 'Не показывать движение врага'.
 # Die Minikarte wird, während die künstliche Intelligenz spielt, unter der Bedingung, dass die Bewegung der künstlicher Intelligenz nicht als „Keine Anzeige der Bewegung der Gegner“, angezeigt.
 # Ο μικρός χάρτης εμφανίζεται όταν είναι η σειρά να παίξει η τεχνητή νοημοσύνη, υπό την προϋπόθεση ότι η κίνηση της τεχνητής νοημοσύνης δεν έχει οριστεί στο «Να μην εμφανίζεται η κίνηση των εχθρών».
-AIMinimap = 1
+AIMinimap = 0
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.